packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts

```diff
@@ -1,11 +1,18 @@
+import { RawTransaction } from '@defichain/jellyfish-api-core/dist/category/rawtx'
```

Comment:
This type isn't an included dependency in `jellyfish-transaction-builder`; it won't be available unless explicitly installed.

Reply:
It also creates an expectation that consumer of this class uses `jellyfish-api-core` to build the transaction. They might be using whale which don't provide this information.